homeassistant/components/light/__init__.py

```diff
@@ -270,7 +265,7 @@ def async_handle_light_service(service):
 
     hass.services.async_register(
         DOMAIN, SERVICE_TOGGLE, async_handle_light_service,
-        descriptions.get(SERVICE_TOGGLE), schema=LIGHT_TOGGLE_SCHEMA)
+        descriptions.get(SERVICE_TOGGLE), schema=LIGHT_TURN_ON_SCHEMA)
```

Comment:
The point is that `SERVICE_TOGGLE` can do everything `SERVICE_TURNON` can do, therefore the `LIGHT_TOGGLE_SCHEMA` is identical to `LIGHT_TOGGLE_SCHEMA`. Do you still want to make a seperate schema for this? Should I do something like the following then?

```python
LIGHT_TOGGLE_SCHEMA = LIGHT_TURN_ON_SCHEMA.extend()
```